### PR TITLE
Added html and css codes. Added all the sections.

### PR DESCRIPTION
**Pull request by daviidy**
Worked on all the requirements given on this project:

Used semantic HTML elements for the header, the main content, the form and the footer.
Styled the elements to look like the real webpage.
Used images as a background and added gradients to elements
Fixed linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+      </ul>
+      <div class="div-search">
+        <i class="fas fa-search"></i>
+        <input type="text" name="search" value="">
+      </div>
+    </nav>
+  </header>
+
+  <article class="">
```
Comment (ghost) [inline index.html]: Remove the empty class on line 81.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    </div>
+  </article>
+  <aside class="promo">
+    <div class="promo-container">
+      <a href="#"> <img class="promo-image" src="./images/promo1.jpg" alt=""></a>
+      <a href="#"> <img class="promo-image" src="./images/promo2.jpg" alt=""></a>
+      <a href="#"> <img class="promo-image" src="./images/promo3.jpg" alt=""></a>
+      <a href="#"> <img class="promo-image" src="./images/promo4.jpg" alt=""></a>
+    </div>
+  </aside>
```
Comment (ghost) [inline index.html]: # Optional
You can wrap these sections inside a `main` tag, since they define the main content of the body.

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required:

## Hello team,

**Good work on your project. I enjoyed reviewing it.**

## There are a few fixes needed before the page can be approved.

# Mandatory

1). **Your page has a horizontal scroll. Please remove it.**

2). **Add a Getting started section on your readme file, with instructions on how to setup your project locally and the necessary tools required. Checkout the [readme template](https://github.com/microverseinc/readme-template) for more on this.**

### Please check the two comments below.

### ps: In case you want to reach out, please write me a message on slack (Rose Wanjohi).

### Good luck! 😃.

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @daviidy AND @sinansevgi 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 Kindly Fix the Semantic HTML warning 👇
- [ ] Fix HTML validation [HTML Validation Screenshort](https://user-images.githubusercontent.com/29689273/93978428-70a46d80-fd95-11ea-9b6d-e1f00ffced02.png). You can check HTML validations in [validator.w3.org](https://validator.w3.org/)



##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — APPROVED** [review]
# **Status: PROJECT APPROVED** :champagne:
Excellent job on this project guys. :fireworks: Keep it up.
##
### The [clone](https://daviidy.github.io/apple-background/) shows a good understanding Backgrounds and Gradients. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
- The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job  @daviidy AND @sinansevgi! :+1: 
- You can now merge to master and submit the project completion form :rocket: . I wish you all the very best in your next tasks :1st_place_medal:
##
**If you have any questions, let me know on slack at any time. I will be happy to help.**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Pull request merged**